### PR TITLE
Cleanup disabled tests missing correct issue numbers

### PR DESCRIPTION
**Pull request by Jiayili1**
1.	[Fact(Skip = "870811")] 
Create a new issue: https://github.com/dotnet/corefx/issues/16747
2.	[Fact(Skip = "906219")]
Create a new issue: https://github.com/dotnet/corefx/issues/16748
Change to [ActiveIssue(issue number)]
3. 	[Fact(Skip = "https://github.com/dotnet/corefx/issues/15101")]
Add Attribute: [ActiveIssue(15101)]
4.   	[ActiveIssue("PerformanceTest")]
Create a new issue: https://github.com/dotnet/corefx/issues/16754 
5.	[ActiveIssue("Valid test but too intensive to enable even in OuterLoop")]
Create a new issue: https://github.com/dotnet/corefx/issues/16750 
6.	[ActiveIssue("Cancellation token not shared")]
Create a new issue: https://github.com/dotnet/corefx/issues/16751 
7.	[ActiveIssue("Fails on desktop and core: 'Unable to cast object of type 'System.UInt32[][*]' to type 'System.Object[]'")]
Create a new issue: https://github.com/dotnet/corefx/issues/16753 
8.	[ActiveIssue("fails when using CodeGen as well")]
Create a new issue: https://github.com/dotnet/corefx/issues/16752 
9.	[ActiveIssue("Distributed transactions are not supported.")]
Create a new issue: https://github.com/dotnet/corefx/issues/16755
10.   [ActiveIssue("dotnet/coreclr#2051", TestPlatforms.AnyUnix)] 
Change to [ActiveIssue("https://github.com/dotnet/coreclr/issues/2051”, TestPlatforms.AnyUnix)].

### CODE REVIEW

**Comment by Jiayili1** [comment]
@Priya91

**Comment by karelz** [comment]
@Jiayili1 it would be easier to split it by area - area owners should review the changes

File: src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ public class AssemblyLoadContextTest
         private const string TestAssembly = "System.Runtime.Loader.Test.Assembly";
 
         [Fact(Skip = "https://github.com/dotnet/corefx/issues/15101")]
```
Comment (danmosemsft) [inline src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs]: Where you have added an ActiveIssue, you no longer need [Fact(Skip="xxx")], just [Fact].

File: src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs [hunk 2]
```diff
@@ -21,7 +21,7 @@ public PerformanceTests(ITestOutputHelper output)
         }
 
         [Theory]
-        [ActiveIssue("PerformanceTest")]
```
Comment (danmosemsft) [inline src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs]: I misspoke about these performance test ones -- they don't need an issue. please change to
```
[Theory(Skip = "Performance test")]
```

They don't need ActiveIssue.

File: src/System.Linq/tests/SelectTests.cs [hunk 3]
```diff
@@ -139,8 +139,8 @@ public void SelectPropertyPassingIndexOnLast()
             Assert.Equal(expected, source.Select((e, i) => i == 5 ? e.name : null));
         }
 
-        [Fact]
-        [ActiveIssue("Valid test but too intensive to enable even in OuterLoop")]
+        [Fact(Skip = "Valid test but too intensive to enable even in OuterLoop"]
+        [ActiveIssue(16750)] //Valid test but too intensive to enable even in OuterLoop
```
Comment (danmosemsft) [inline src/System.Linq/tests/SelectTests.cs]: Please put one space after `//`

**Comment by danmosemsft** [comment]
Please update per feedback (where an issue is in several places I only noted one)

You see you have a merge conflict with 'enlisttest.cs' - to fix this please rebase -- you can find documentation online, but it's basically
```
git checkout master
git pull
git checkout  AuditActiveIssues
git rebase master
git push --force
```

**Comment by danmosemsft** [comment]
@karelz I thought for this change it's pretty uncontroversial and does not need wide review.

**Comment by karelz** [comment]
@danmosemsft yeah, you're probably right (I already edited my comment earlier with questioning myself).
BTW: It might help to educate affected areas on the "right way", but it is lower pri.

File: src/System.Linq/tests/SelectManyTests.cs [hunk 4]
```diff
@@ -212,8 +212,8 @@ public void IndexCausingLastToBeSelected()
             Assert.Equal(source.Last().total, source.SelectMany((e, i) => i == 4 ? e.total : Enumerable.Empty<int?>()));
         }
 
-        [Fact]
-        [ActiveIssue("Valid test but too intensive to enable even in OuterLoop")]
+        [Fact(Skip = "Valid test but too intensive to enable even in OuterLoop"]
```
Comment (danmosemsft) [inline src/System.Linq/tests/SelectManyTests.cs]: I gave bad guidance for these "too intensive" ones -- I think we do not need `[ActiveIssue]` as there is no bug. Just ` [Fact(Skip = "Valid test but too intensive to enable even in OuterLoop"]`.
I will close #16750

Comment (Priya91) [inline src/System.Linq/tests/SelectManyTests.cs]: Missing close brace, this won't compile..

File: src/System.Linq/tests/SelectTests.cs [hunk 5]
```diff
@@ -139,8 +139,8 @@ public void SelectPropertyPassingIndexOnLast()
             Assert.Equal(expected, source.Select((e, i) => i == 5 ? e.name : null));
         }
 
-        [Fact]
-        [ActiveIssue("Valid test but too intensive to enable even in OuterLoop")]
+        [Fact(Skip = "Valid test but too intensive to enable even in OuterLoop"]
```
Comment (Priya91) [inline src/System.Linq/tests/SelectTests.cs]: Missing close brace, this won't compile..

File: src/System.Linq/tests/SkipWhileTests.cs [hunk 6]
```diff
@@ -141,9 +141,9 @@ public void AllButLastExcludedByIndex()
 
             Assert.Equal(expected, source.SkipWhile((element, index) => index < source.Length - 1));
         }
-        
-        [Fact]
-        [ActiveIssue("Valid test but too intensive to enable even in OuterLoop")]
+
+        [Fact(Skip = "Valid test but too intensive to enable even in OuterLoop"]
+        [ActiveIssue(16750)] //Valid test but too intensive to enable even in OuterLoop
```
Comment (Priya91) [inline src/System.Linq/tests/SkipWhileTests.cs]: Same as before, please checkout other instances below as well.

File: src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs (hunk 2 above)
Comment (Priya91) [inline src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs]: We mark performance tests with ```[Benchmark]``` attribute, so that these can be run with ```-trait Benchmark=true```.

File: src/System.Linq/tests/SelectTests.cs (hunk 5 above)
Comment (danmosemsft) [inline src/System.Linq/tests/SelectTests.cs]: @Jiayili1 I may mave missed this from the steps, but before submitting a PR it's wise to build the tree from the root with "build.cmd". You can also build an individual project with "msbuild <projectname>.csproj".

The CI results here will show the build errors, but it takes a lot of time to get them back when building locally would find issues in a moment.

**Comment by Priya91** [comment]
> You see you have a merge conflict with 'enlisttest.cs' - to fix this please rebase -- you can find documentation online, but it's basically
> git checkout master
> git pull
> git checkout  AuditActiveIssues
> git rebase master
> git push --force

Or will on AuditActiveIssues branch you can just do,

```
git pull --rebase https://github.com/dotnet/corefx.git master
```

File: src/System.Linq/tests/SelectManyTests.cs (hunk 4 above)
Comment (JonHanna) [inline src/System.Linq/tests/SelectManyTests.cs]: I'm not sure if these are worth keeping at all.

Comment (danmosemsft) [inline src/System.Linq/tests/SelectManyTests.cs]: @VSadov @OmarTawfik ?

File: src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs (hunk 2 above)
Comment (danmosemsft) [inline src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs]: @Priya91 how do we skip in runs, yet not actually disable them (so -trait would make them run on demand).

Comment (Priya91) [inline src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs]: ```[Benchmark]``` tests are skipped by default in CI. Build scripts pass ```-notrait Benchmark=true```. So these can only be run on demand by setting xunit Benchmark trait.

Comment (danmosemsft) [inline src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs]: OK @Jiayili1 to summarize, for the PerformanceTest ones, they should be just 
```
[Theory]
[Benchmark]
```

and no ActiveIssue of course.

Comment (Priya91) [inline src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs]: We don't need [Theory], just [Benchmark], you can refer example [here](https://github.com/dotnet/corefx/blob/master/src/System.Text.Encoding/tests/Performance/Perf.Encoding.cs#L23)

File: src/System.Collections/tests/Generic/SortedSet/SortedSet.Generic.Tests.cs [hunk 7]
```diff
@@ -351,14 +351,5 @@ public void SetComparer_SetEqualsTests()
             Assert.True(comparerSet2.SetEquals(set));
         }
         #endregion
-
```
Comment (danmosemsft) [inline src/System.Collections/tests/Generic/SortedSet/SortedSet.Generic.Tests.cs]: These sortedset changes are unrelated to your change and in fact were made by @stephentoub. I'm not sure how you managed to rebase such that they are showing up as a diff. I guess they will merge away.

**Comment by danmosemsft** [comment]
LGTM thanks

**Comment by danmosemsft** [comment]
@Jiayili1 you can now click "delete branch" above, and also delete the local branch with 
```
git checkout master
git pull
git remote prune origin
git branch -d AuditActiveIssues
```
